modules/questdb/src/main/java/org/testcontainers/containers/QuestDBContainer.java

```diff
@@ -0,0 +1,85 @@
+package org.testcontainers.containers;
+
+import lombok.NonNull;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Testcontainers implementation for QuestDB.
+ */
+public class QuestDBContainer extends JdbcDatabaseContainer<QuestDBContainer> {
+
+    static final String DATABASE_PROVIDER = "postgresql";
+
+    private static final String DEFAULT_DATABASE_NAME = "qdb";
+
+    private static final int DEFAULT_COMMIT_LAG_MS = 1000;
+
+    private static final String DEFAULT_USERNAME = "admin";
+
+    private static final String DEFAULT_PASSWORD = "quest";
+
+    private static final Integer POSTGRES_PORT = 8812;
+
+    private static final Integer REST_PORT = 9000;
+
+    private static final Integer ILP_PORT = 9009;
+
+    static final String TEST_QUERY = "SELECT 1";
+
+    static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("questdb/questdb");
+
+    public QuestDBContainer(@NonNull String dockerImageName) {
+        this(DockerImageName.parse(dockerImageName));
+    }
+
+    public QuestDBContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+        withExposedPorts(POSTGRES_PORT, REST_PORT, ILP_PORT);
+        withCommitLag(DEFAULT_COMMIT_LAG_MS);
```

Comment:
Why did you choose this commit lag value as the default for the module? What is the difference from the default value in the image?

Reply:
hello 👋 I can comment on this. 

Short answer:
`commit.lag` delays the visibility of ingested rows. Default `commit.lag` is 5 minutes. It was chosen for production env., but it's not really suitable for testing. You do not want to wait 5 minutes for your data to become visible so a test can pass. 

Longer answer:
It's related to QuestDB data organization. Data on disk are physically sorted by time. That's an invariant. When we receive an entry with a timestamp older than the newest entry already on a disk then QuestDB triggers a Copy-On-Write merge. This is be expensive and it's desirable to minimize it. So QuestDB accumulates received rows in memory and persists them to a disk only after some delay (+some other conditions). Here are gory details: https://questdb.io/blog/2021/05/10/questdb-release-6-0-tsbs-benchmark/#how-we-sort-merge-and-commit-out-of-order-time-series-data

Reply:
Thanks for providing the detailed context @jerrinot, much appreciated 🙏 

I think it is great to set it to a value that is optimized for testing and at the same time I think, we don't need to expose it as a public API. Thanks for changing it accordingly @Vangreen.